lib/components/WheelPicker/index.js

```diff
@@ -2,11 +2,26 @@ import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React, {Component} from 'react';
 import {DeviceEventEmitter, requireNativeComponent, StyleSheet, View} from 'react-native';
-import {Picker} from '@react-native-community/picker';
+
 import WheelPickerItem from './WheelPickerItem';
 import {Constants} from '../../../src/helpers';
 import {Typography, Colors} from '../../../src/style';
 
+/* Safely require picker component */
+let Picker = () => null;
+try {
+  Picker = require('@react-native-picker/picker').Picker; // New package
+} catch (e) {
+  try {
+    Picker = require('@react-native-community/picker').Picker; // Deprecated package
+    console.warn(`RNUILib Picker will soon migrate to use "@react-native-picker/picker" package instead of '@react-native-community/picker'`);
+  } catch (e) {
+    if (Constants.isIOS) {
```

Comment:
This `if` can surround the whole section (the `try` `catch`).

Reply:
Fixed